dev/com.ibm.ws.microprofile.reactive.messaging.kafka/src/com/ibm/ws/microprofile/reactive/messaging/kafka/KafkaInput.java

```diff
@@ -213,6 +206,28 @@ public void shutdown() {
         return result;
     }
 
+    /**
+     * Run any pending actions and then poll Kafka for messages.
+     * Return false if a WakeupException was caught, otherwise return true.
```

Comment:
Originally, I was going to suggest changing the javadoc to:
`@return @{code true} if @{code result} was completed, either with records from kafka, or with a fatal exception. @{code false} if @{code result} was not completed because @{code wakeup} was called.`

However, maybe it would make more sense to move the `while()` loop inside this method and remove the return value entirely?

Reply:
Done